Libraries/Components/View/ReactNativeViewAttributes.js

```diff
@@ -45,4 +45,8 @@ ReactNativeViewAttributes.RCTView = {
   removeClippedSubviews: true,
 };
 
+ReactNativeViewAttributes.RCTTVView = {
```

Comment:
I'd avoid redeclaring this here, and just use `ReactNativeViewAttributes.RCTView` in TVView.ios.js